internal/log/log.go

```diff
@@ -19,13 +19,17 @@ import (
 )
 
 // Level specifies the logging level that the log package prints at.
-type Level int
+type Level string
 
 const (
 	// LevelDebug represents debug level messages.
-	LevelDebug Level = iota
-	// LevelWarn represents warning and errors.
-	LevelWarn
+	LevelDebug Level = "DEBUG"
+	// LevelInfo represents info level messages.
+	LevelInfo Level = "INFO"
+	// LevelWarn represents warning messages.
+	LevelWarn Level = "WARN"
+	// LevelError represents error messages.
+	LevelError Level = "ERROR"
 )
```

Comment:
What's the reason for declaring the log levels as strings rather than ints? IMO this makes the severity of the levels more ambiguous. I'd prefer an int type with a String() method.